common/api-review/app-exp.api.md

```diff
@@ -35,11 +33,57 @@ export const _DEFAULT_ENTRY_NAME = "[DEFAULT]";
 // @public
 export function deleteApp(app: FirebaseApp): Promise<void>;
 
-export { FirebaseApp }
-
-export { FirebaseAppConfig }
+// @public
+export interface FirebaseApp {
+    automaticDataCollectionEnabled: boolean;
+    readonly name: string;
+    readonly options: FirebaseOptions;
+}
+
+// @public (undocumented)

```

Comment:
Yup. Added comments for all except for the members of `FirebaseOptions` since they are supposed to be opaque to developers and are kinda self explanator